mlx_graphs/nn/conv/gcn_conv.py

```diff
@@ -76,3 +75,11 @@ def message(
     def _degree(self, index: mx.array, num_edges: int) -> mx.array:
         one = mx.ones((index.shape[0],))
         return scatter(one, index, num_edges, "add")
+
+    def _deg_inv_sqrt(deg: mx.array) -> mx.array:
+        deg_type = {mx.float16: 1e-6, mx.float32: 1e-10}
+        deg += deg_type(deg.dtype)
```

Comment:
I don't know how this can work since your do `deg_type(deg.dtype)` instead of `deg_type[deg.dtype]`. Did you run tests?